accounts.go

```diff
@@ -57,21 +59,29 @@ func ParseAccountsMetrics(input []byte) map[string]*JobMetrics {
                         account := strings.Split(line,"|")[1]
                         _,key := accounts[account]
                         if !key {
-                                accounts[account] = &JobMetrics{0,0,0,0,0}
+                                accounts[account] = &JobMetrics{0,0,0,0,0,0,0}
                         }
                         state := strings.Split(line,"|")[2]
                         state = strings.ToLower(state)
                         cpus,_ := strconv.ParseFloat(strings.Split(line,"|")[3],64)
+                        gres := strings.Split(line, "|")[4]
+                        var gpus float64 = 0
+                        if gres != "N/A" {
+                                gpus, _ = strconv.ParseFloat(gres[9:], 64)
```

Comment:
Question what does gres look like?
Will this break for multinode or large GPU requests?

Reply:
On our side it always looks like gres:gpu:8 or some other number. It isn't impacted by the number of nodes or the like. It is also manually setup in the slime conf by us: https://slurm.schedmd.com/gres.html, so won't be impacted by a spurn update in all likelihood.

If we change how we do gpu accounting we will have to update this. We would also have to change the original gpu accounting the Prometheus had because it uses the same method of extracting gpu count.

Reply:
Okily dokily